Help/FAQ.rst

```diff
@@ -8,23 +8,21 @@ What does coala do (for me)?
 ----------------------------
 
 coala is like a spell and grammar checker for source code. Imagine using
-LibreOffice Writer to spellcheck something in English. Why should you have to
-write LibreOfficES to check something written in Spanish?
-That is what coala  tries to fix. Why write the whole thing again if you
-actually just want another analyzing routine? Thats what bears are for. You
-want to use clang or pylint on your project? We got you covered. One command
+LibreOffice Writer to spellcheck something in English but to spellcheck
+something in Spanish is pretty different. That is what coala  tries to fix.
```

Comment:
" is pretty different."  this doesnt capture the same meaning as "LibreOfficES".